database/TenMillionData.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+
+const url = `mongodb+srv://Henry:henry@cluster0.8a9be.mongodb.net/airbnb?retryWrites=true&w=majority`;
+
+mongoose.connect(url, {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false
+});
+
+const connection = mongoose.connection;
+
+connection.once('open', () => {
+  console.log('MongoDB database connection established successfully');
+});
+
+const randomInt = function (max) {
+  return Math.floor(Math.random() * max);
+};
+
+var randomPrice = function () {
+  return Math.floor(Math.random() * 57) * 100 + 400 - Math.floor(Math.random() * 2);
+};
+
+const prefixes = [
+  'The', 'Wonderful', 'Magnificent', 'Quiet', 'Gorgeous', 'Relaxing', 'Luxury',
+  'Ancient', 'Charming', 'Fascinating', 'Homey', 'Peaceful', 'Magical'];
+
+const locations = [
+  () => { return faker.address.country() + ' style'; },
+  () => { return faker.address.city(); },
+  () => { return faker.address.state(); },
+  () => { return faker.name.firstName(); }
+];
+
+const housingTypes = [
+  'Villa', 'Palace', 'Bungalow', 'Mansion', 'Escape', 'House',
+  'Vista', 'Resort', 'Hotel'];
+
+const randomLocationName = () => {
+  var name = `${prefixes[randomInt(13)]} ${locations[randomInt(4)]()} ${housingTypes[randomInt(9)]}`;
+  return (name.length < 36) ? name : randomLocationName();
+};
+
+const documents = [];
+const totalNumberOfData = 10000000;
+
+for (var i = 1; i <= totalNumberOfData; i++) {
+  const document = { id: i};
+
+  document.name = randomLocationName();
+
+  document.price = randomPrice();
+
+  document.imageurl = `https://airbnbvillas.s3.us-east-2.amazonaws.com/${randomInt(28)+1}.jpg`;

```

Comment:
Similar problem here, although not that big of a deal. I would pull the S3 bucket URL from env variables / uncommitted config.js file.